mail-client/alpine/alpine-2.26.ebuild

```diff
@@ -43,7 +43,6 @@ src_configure() {
 		$(use_with ssl)
 		$(use_with passfile passfile .pinepwd)
 		$(use_with kerberos krb5)
-		$(use_with threads pthread)
```

Comment:
Could you just hardcode it anyway, so the configure option is visible to everyone? There are some hardcoded values above these conditional ones in the ebuild.

`--use-with-pthread`.

Reply:
That may be wrong for other non-Linux platforms, but I agree in principle with doing this for any option where we can.

Reply:
Looking at it, it looks like pthread is fine here though.

Reply:
This is a standalone application so it does not affect anything else. The real problem is with libraries. Also this is just a remnant for compatibility with *ahem* crowd pleasers like AIX and HP-UX.